src/asfvideo.cpp

```diff
@@ -83,13 +83,13 @@ std::string AsfVideo::GUIDTag::to_string() {
 bool AsfVideo::GUIDTag::operator<(const GUIDTag& other) const {
   if (data1_ < other.data1_)
     return true;
-  else if (data1_ == other.data1_) {
+  if (data1_ == other.data1_) {
     if (data2_ < other.data2_)
       return true;
-    else if (data2_ == other.data2_) {
+    if (data2_ == other.data2_) {
       if (data3_ < other.data3_)
```

Comment:
Would you mind to take this opportunity to add the brackets were they are missing? It looks super confusing to have some ifs/elses with them and other without.